app/components/avatar_component.rb

```diff
@@ -9,7 +9,7 @@ class AvatarComponent < ApplicationComponent
 
   def avatar
     if image_url
-      image_tag(url(image_url), alt: alt, class: image_classes)
```

Comment:
This is causing an error in production 
```
NoMethodError (undefined method `url' for an instance of AvatarComponent)
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     18:                 <div class="d-flex align-items-end justify-content-between">
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     19:                   <div class="d-flex align-items-center">
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     20:                     <div class="me-2 position-relative d-flex justify-content-end align-items-end mt-n5">
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     21:                       <%= render AvatarComponent.new(current_user, size: :xl) %>
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     22:                     </div>
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     23:                     <div class="lh-1">
[2ea121ff-ad45-4bc7-b5f3-5ceb98d47fdf]     24:                       <h2 class="mb-0"><%= "#{current_user.full_name}" %></h2>
```

Getting rid of `url` method works fine locally, and I think will fix this issue in production.